app/helpers/api_keys_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ApiKeysHelper
+  def mask_api_key(api_key)
```

Comment:
Maybe to match the other methods here you could pass the whole object? That would also ensure the correct value is masked.